### ISSUE TITLE
Heading state remains when changed to Paragraph in WYSIWYG mode.

### ISSUE THREAD

**Issue by ang-d** [nhn/tui.editor#1971]
## Describe the bug
When using the toolbar to change heading level from Heading > 0 to Paragraph, the `H` in toolbar remains blue, indicating heading state is still triggered for the line.

This causes the line to not execute lists commands, etc. due to be in heading state.

## To Reproduce
1. In tui-editor demo, in WYSIWYG mode, change a heading 1 to paragraph using the toolbar menu.
2. Observe line is styled as paragraph but toolbar still indicates Heading state.
3. Click on unordered list, it will not work.
4. Switch to Markdown mode and back to WYSIWYG mode, issue resolved.

## Expected behavior
- Heading state should not remain when chose Paragraph from Heading menu.

## Screenshots
![image](https://user-images.githubusercontent.com/55774873/139569719-146acc1c-30ad-4e56-ac57-c986228020fb.png)

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser Chrome, Electron

**Comment by ang-d** [comment]
Current workaround, not sure if this is the best method but hope this helps:
- under `Heading.prototype.commands`
  ```js
  var nodeType = payload.level > 0
    ? state.schema.nodes.heading
    : state.schema.nodes.paragraph;
  return setBlockType(nodeType, payload)(state, dispatch);
  ```

**Comment by js87zz** [comment]
@ang-d 
Sorry for late replying.
I checked issue and it will be fixed. Thanks!

**Comment by js87zz** [comment]
@ang-d 
The bug has been fixed in `v3.1.2`. Thanks!
